keyboards/handwired/wwa/rosetta/readme.md

```diff
@@ -0,0 +1,27 @@
+# rosetta
+
+![rosetta](https://i.imgur.com/KQL8cUe.jpg)
+
+21 Keys, 4 Fader MIDI Macro Keyboard
```

Comment:
Do you intend to include any support for the sliders in the firmware?

Reply:
Not at the moment, i'm still trying to figure out the code. Will commit the changes later on

Reply:
> Will commit the changes later on

On that note, could I ask, to save hassle in the long term, this PR is set to Draft until such time that said code has been added?

Reply:
Just committed the MIDI code, it is working, please let me know if this is the right way of doing it, if not, i'll make the changes accordingly

Reply:
[MIDI Documenation](https://docs.qmk.fm/#/feature_midi) good first port of call 😉